### PR TITLE
feat: NgBootstrap removal documentation

### PR DESCRIPTION
**Pull request by pawelfras**

### CODE REVIEW

File: _pages/home/updating-to-version-5/updating-to-version-5.md [hunk 1]
```diff
@@ -70,10 +79,94 @@ With the release of Spartacus 5.0, the hosting service of SAP Commerce Cloud can
 
    For more information, see [Adding Applications for JavaScript Storefronts](https://help.sap.com/docs/SAP_COMMERCE_CLOUD_PUBLIC_CLOUD/b2f400d4c0414461a4bb7e115dccd779/63577f67a67347bf9f4765a5385ead33.html).
 
+## Removal of ng-bootstrap
+
+ng-bootstrap has been removed from Spartacus 5.0 and is no longer a dependency. You can continue to use ng-bootstrap in your Spartacus application, but this may require upgrading additional third party dependencies. For more information, see the list of [ng-bootstrap dependencies](https://www.npmjs.com/package/@ng-bootstrap/ng-bootstrap).
```
Comment (Matejk00) [inline _pages/home/updating-to-version-5/updating-to-version-5.md:84]: Nitpick :) 

Maybe let's consider to name change to NgBootstrap or NG Bootstrap as it is name on https://www.npmjs.com/package/@ng-bootstrap/ng-bootstrap

ng-bootstrap refers more to package name.

What do you think?

File: _pages/home/updating-to-version-5/updating-to-version-5.md [hunk 2]
```diff
@@ -70,10 +79,94 @@ With the release of Spartacus 5.0, the hosting service of SAP Commerce Cloud can
...
+
+If you no longer need ng-bootstrap in your Spartacus application, you can run the command `yarn remove @ng-bootstrap/ng-bootstrap`.
+
+## Removal of ModalService and Related Code
+
+The `ModalService`, `ModalDirective`, `ModalDirectiveOptions`, `ModalDirectiveService`, `ModalOptions` and `ModalRef` have all been removed from Spartacus 5.0. If you are using the `ModalService` and related code in your Spartacus application, you can replace it with the `LaunchDialogService` provided by Spartacus.
```
Comment (Matejk00) [inline _pages/home/updating-to-version-5/updating-to-version-5.md:90]: Maybe let's add a package name. That `LaunchDialogService` can be imported from `@spartacus/storefront`.

File: _pages/home/updating-to-version-5/updating-to-version-5.md [hunk 3]
```diff
@@ -70,10 +79,94 @@ With the release of Spartacus 5.0, the hosting service of SAP Commerce Cloud can
...
+Once you have a key for your component, you need to provide a configuration to the component module that is specified for your modal. The following is an example:
+
+```ts
+import { DIALOG_TYPE, LayoutConfig } from '@spartacus/storefront';
+import { CloseAccountModalComponent } from './close-account-modal.component';
+
+export const defaultCloseDialogModalLayoutConfig: LayoutConfig = {
+    launch: {
+       CLOSE_ACCOUNT: {
+          inline: true,
```
Comment (Matejk00) [inline _pages/home/updating-to-version-5/updating-to-version-5.md:125]: Should we also mention about other rendering strategies like `inlineRoot` for example?

**Review by Matejk00 — CHANGES_REQUESTED** [review]

File: _pages/home/updating-to-version-5/updating-to-version-5.md (hunk 1 above)
Comment (pawelfras) [reply _pages/home/updating-to-version-5/updating-to-version-5.md:84]: Good catch! I'll definitely change this name to NG Bootstrap 👌

**Review by Matejk00 — APPROVED** [review]

**Review by gladius-mtl — APPROVED** [review]

**Pull request merged**